ospf6d/ospf6_top.c

```diff
@@ -880,6 +881,24 @@ DEFUN (no_ospf6_log_adjacency_changes_detail,
 	return CMD_SUCCESS;
 }
 
+DEFPY (ospf6_send_extra_data,
+       ospf6_send_extra_data_cmd,
+       "[no] ospf6 send-extra-data zebra",
+       NO_STR
+      OSPF6_STR
+       "Extra data to Zebra for display/use\n"
+       "To zebra\n")
+{
+	VTY_DECLVAR_CONTEXT(ospf6, ospf6);
+
+        if (no)
+		UNSET_FLAG(ospf6->config_flags, OSPF6_SEND_EXTRA_DATA_TO_ZEBRA);
```

Comment:
If the extra data is already sent to zebra and then this command disables it, it is expected to keep these flags as it is with zebra? Or if was disabled and enabled later then what would happen to routes which were already sent to zebra?

Reply:
I've updated the code to reinstall all OSPF routes once opaque attributes are enabled or disabled.